src/internal/parallel/each.go

```diff
@@ -1,7 +1,12 @@
 package parallel
 
+// Action is a function that returns an error, this is used by ForEach
```

Comment:
I've put some thought into this and I want to avoid the word `interface`

How do you feel about 

```go
// Action is the function to be consumed by ForEach. It should return an error if the action failed
// and should not be retried
```

Reply:
Sounds good!